openshift/README.md

```diff
@@ -213,13 +214,42 @@ oc rollout -n $NAMESPACE latest dc/<buildname>-master
 
 *Note: Remember to swap out the bracketed values with the appropriate values!*
 
-### Sidecar Logging
+## Sidecar Logging
 
 Our deployment on OpenShift uses a Fluent-bit sidecar to collect logs from the CHEFS application. The sidecar deployment is included in the main app.dc.yaml file.
 Our NodeJS apps output logs to a configurable file path (for example app/app.log ). This is done using using a logger script. For example see our [CHEFS app logger](https://github.com/bcgov/common-hosted-form-service/blob/master/app/src/components/log.js)
 
+The Fluent-bit configuration is kept in the openshift config map [fleunt-bit.cm.yaml](https://github.com/bcgov/common-hosted-form-service/blob/master/openshift/fluent-bit.cm.yaml)
+
 Additional details for configuring the sidecar can be seen on the [wiki](https://github.com/bcgov/nr-get-token/wiki/Logging-to-a-Sidecar).
 
+### Logs sent to AWS Opensearch
+
+We currently forward our application logs from Fluent-bit to an AWS OpenSearch service.
+the AWS connection credentials are found using environment variables in the fluent-bit container (aws credentials stored in 'chefs-aws-kinesis-secret' secret.)
+
+to create this secret on OpenShift:
+
+```sh
+export NAMESPACE=<yournamespace>
+export APP_NAME=<yourappshortname>
+
+export username=<AWS access key ID>
+export password=<AWS secret access key>
+
+oc create -n $NAMESPACE secret generic $APP_NAME-aws-kinesis-secret \
+  --type=kubernetes.io/basic-auth \
+  --from-literal=username=$username \
+  --from-literal=password=$password
+```
+
+The Fluent-bit configuration includes the output plugin 'kinesis streams' where we define our AWS region, arn_role and stream name.
+A further parser for our logs was added to a node app running on an [AWS Lambda service](https://github.com/BCDevOps/nr-elasticsearch-stack/tree/master/event-stream-processing/src)
+
+### Error Notifications
+
+TODO: We currently also output logs to a Fluentd service where we can trigger error notifications to our Discord channel. See our [Wiki](https://github.com/bcgov/nr-get-token/wiki/Aggregate-logs-with-Fluentd) from more details. In due course, we can try the notification tools available in the AWS OpenSearch service.
```

Comment:
I'd personally drop the last sentence in this paragraph, and unflag this as a TODO, as alerting and error notifications are outside the scope of this repo's concern.